README.md

```diff
@@ -192,134 +89,30 @@ local on_attach = function(client, bufnr)
-    can optionally use {filetype} to help pre-filter by filetype.
-
-    If a root_dir is returned which is unique from any previously returned
-    root_dir, a new server will be spawned with that root_dir.
-
-    If nil is returned, the buffer is skipped.
-
-    See |lspconfig.util.search_ancestors()| and the functions which use it:
-    - |lspconfig.util.root_pattern(pattern1, pattern2...)| is a variadic function which
-      takes string patterns as arguments, and finds an ancestor 
-      which contains one of the files matching the pattern. 
-      Each pattern can be a specific filename, such as ".git", or a glob.  
-      See `:help glob` for allowed patterns.  This is equivalent to
-      coc.nvim's "rootPatterns"
-    - Related utilities for common tools:
-      - |lspconfig.util.find_git_root()|
-      - |lspconfig.util.find_node_modules_root()|
-      - |lspconfig.util.find_package_json_root()|
-
-  {name}
-    Defaults to the server's name.
-
-  {filetypes}
-    Set of filetypes to filter for consideration by {root_dir}.
-    May be empty.
-    Server may specify a default value.
-
-  {autostart}
-    Whether to automatically start a language server when a matching filetype is detected.
-    Defaults to true.
-
-  {settings}
-    Map with case-sensitive keys corresponding to `workspace/configuration`
-    event responses.
-    We also notify the server *once* on `initialize` with
-    `workspace/didChangeConfiguration`.
-    If you change the settings later on, you must emit the notification
-    with `client.workspace_did_change_configuration({settings})`
-    Example: `settings = { keyName = { subKey = 1 } }`
-
-  {on_attach}
-    `function(client, bufnr)` Runs the on_attach function from the client's 
-    config if it was defined. Useful for doing buffer-local setup.
-
-  {on_new_config}
-    `function(new_config, new_root_dir)` will be executed after a new configuration has been
-    created as a result of {root_dir} returning a unique value. You can use this
-    as an opportunity to further modify the new_config or use it before it is
-    sent to |vim.lsp.start_client()|. If you set a custom `on_new_config`, ensure that 
-    `new_config.cmd = cmd` is present within the function body.
-```
+The `on_attach` hook is used to only activate the bindings after the language server attaches to the current buffer.
 
 ## Debugging
-The two most common points of failure are 
+The two most common reasons a language server does not start or attach are:
 
-1. The language server is not installed. You should be able to run the `cmd` 
-defined in each lua module from the command line.
+1. The language server is not installed. nvim-lspconfig does not install language servers for you. You should be able to run the `cmd` defined in each server's lua module from the command line and see that the language server starts. If the `cmd` is an excutable name, ensure it is on your path.
```

Comment:
Perhaps the README should ask users to run `:checkhealth` since it reports whether a command is executable?

Reply:
No worries!